proto-lens-tests/tests/imports_test.hs

```diff
@@ -59,3 +62,16 @@ testUseBootstrapped :: TestTree
 testUseBootstrapped = testCase "testUseBootstrapped" $ do
     testField @Imports.UseBootstrapped @Descriptor.DescriptorProto #descriptor
     testField @Imports.UseBootstrapped @Plugin.CodeGeneratorRequest #request
+
+testDescriptor :: TestTree
```

Comment:
I  wasn't sure whether having this test here makes sense, or whether I should make a new test just for this functionality.

Reply:
Let's make a new test. (Maybe something like `message_test` or `descriptor_test`?)  It seems like no existing test is a good fit for this, and it's generally better not to use the same sample "data" for two different uses.  (Also, in this case, transitive_dep.proto comes from a different test package, making it a little more complicated than needed.)